### ISSUE TITLE
Assets upload showing error if uploading more than one image

### ISSUE THREAD

**Issue by redred** [statamic/cms#2500]
## Bug Description

I created a FieldSet with an Assets field in the CMS and when I add to the collection and attempt to load more than one image, I get an error saying:

`Call to a member function allowUploads() on null`

<img width="325" alt="Screen Shot 2020-09-22 at 11 56 54 AM" src="https://user-images.githubusercontent.com/413592/93910098-9dda1880-fcce-11ea-8d24-13d7ea5531aa.png">

## How to Reproduce

Create an Assets field and upload more than one image and you should see the error. Additionally, if you upload one image it will work fine, but if you attempt to upload a second image without first saving the entry, the same error appears.

## Extra Detail

It seems that if I save the entry, the page reloads and uploads begin to work again. So it seems reloading the page does something. I also tried clicking on save and continue editing at the top, but that does not allow uploading again.

## Environment

I'm running Statamic locally on a MacBook Pro on Catalina 10.15.6, using MAMP Pro with Apache 2.2, PHP 7.3.1.

**Statamic version:** 3.0.{?}
Statamic version is 3.0.10

**PHP version:** 7.{?}
PHP 7.3.1

**Install method (choose one):**
This is a fresh install from `statamic/statamic`

**Comment by jasonvarga** [comment]
Possibly a duplicate of #1663

**Comment by redred** [comment]
Per the 1663 link by @jasonvarga setting Container to Assets seems to allow at least two uploads at a time. Thanks.

<img width="477" alt="Screen Shot 2020-09-22 at 2 14 34 PM" src="https://user-images.githubusercontent.com/413592/93921372-7723de00-fcde-11ea-8c68-c1f1658ad00a.png">

**Comment by damcclean** [comment]
I've just added an Asset field to a blueprint. When I Ieft the field without a container and tried to create an entry, I got an exception telling me to set an asset container to the field.

However, after I added an asset container to the field, I didn't get any exceptions and multiple image uploads worked fine.

I'm closing this issue now as it seems you're no longer able to have an asset field without a container being set.
